rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.74.0"
+channel = "nightly-2023-12-01"
```

Comment:
Should probably not change toolchain all the time?

Reply:
Should have it in my .gitignore, sry